docs: add release notes (#1741)
100mslive/100ms-docs: docs/javascript/v2/changelog/release-notes.mdx

| @100mslive/hls-player                  | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhls-player.svg)](https://badge.fury.io/js/%40100mslive%2Fhls-player)           |
| @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |

## 2023-08-25

> Released: `@100mslive/hms-video-store@0.10.14`, `@100mslive/react-sdk@0.8.14`, `@100mslive/hls-player@0.1.14`

Reviewer: ```suggestion
Released: `@100mslive/hms-video-store@0.10.14`, `@100mslive/react-sdk@0.8.14`, `@100mslive/hls-player@0.1.14`, `@100mslive/roomkit-react@0.1.5`
```

Fixed: 
- Audio of some remote peers not audible after role change from a subscribing role to a non-subscribed role and coming back to a subscribing role

## 2023-08-24

> Released: `@100mslive/hms-video-store@0.10.13`, `@100mslive/react-sdk@0.8.13`, `@100mslive/hls-player@0.1.13`

Reviewer: should add roomkit-react version as well

> Released: `@100mslive/hms-video-store@0.10.13`, `@100mslive/react-sdk@0.8.13`, `@100mslive/hls-player@0.1.13`

Reviewer: ```suggestion
Released: `@100mslive/hms-video-store@0.10.13`, `@100mslive/react-sdk@0.8.13`, `@100mslive/hls-player@0.1.13`, `@100mslive/roomkit-react@0.1.4`, `@100mslive/react-icons@0.8.13`
```

Fixed: 
- Reduced bundle size for @100mslive/roomkit-react and @100mslive/react-icons

> - New design changes and bug fixes in Prebuilt Component

Reviewer: ```suggestion
- New design changes and bug fixes in `HMSPrebuilt` Component of `roomkit-react` package
```
